### PR TITLE
Log: Replace namespace log with logging

### PR DESCRIPTION
**Pull request by wjwithagen**
- This to avoid a naming conflict that Clang finds with
   std::log from cmath

Signed-off-by: Willem Jan Withagen <wjw@digiware.nl>

### CODE REVIEW

File: src/common/Graylog.cc [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #include "include/uuid.h"
 
 namespace ceph {
-namespace log {
+namespace loging {
```
Comment (tchaikov) [inline src/common/Graylog.cc]: s/loging/logging/

**Comment by wjwithagen** [comment]
@tchaikov 
Redid the patch since I used loging instead of logging. But it was easy since I just fixed it in the patchfile and reapplied the patch.
Lets see if Jenkins likes it.
I'm off for the evening, so I'll see the result when I return.

**Comment by wjwithagen** [comment]
@tchaikov 
I guess that this is going to take a full QA run before it can be merged? And thus will take some time.?
Do you know if any developer info/doc needs to be changed if this PR would be merged?

**Comment by tchaikov** [comment]
this change looks good to me.

@liewegas is it good to merge if it builds on gitbuilder and the qa run passes?

**Comment by tchaikov** [comment]
@wjwithagen i don't think there is any dev doc is referencing the changed namespace "log". maybe what we need is just a bless from another developer.

**Comment by wjwithagen** [comment]
@liewegas @cbodley  
Clang trips over the use of namespace log. Most likely due to the loading of the ceph namespace t global level. @tchaikov has looked into removing this, and rewriting references as needed. Which turns out a lot of trivial changes in many files. 
So the alternative is to avoid this naming conflict and rename the namespace.
Which turns out to be a rather limited change. 
Would you care to take a look at this and bless it?

**Comment by liewegas** [comment]
Yep, lgtm :)

**Comment by wjwithagen** [comment]
@liewegas 
Thanx, and waiting for QA to do it thing.

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2016-10-29_03:47:14-rados-wip-kefu-testing---basic-mira/

**Comment by wjwithagen** [comment]
@tchaikov 
Thanx for helping.

**Pull request merged**